Enterprise-CMCS/eAPD
# NZ/4441 feature temporary message

Resolves #4441 

### Description
On the successful creation of an APD, a temporary message is displayed saying that their APD has been created and a small instruction how to go through the APD.

![Screen Shot 2022-12-20 at 10 55 23 AM](https://user-images.githubusercontent.com/44708048/208710036-1b8f95b8-f00b-41ba-b04e-3cb368a658b3.png)

### Significant changes or possible side effects

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| An APD | created | shows a temporary success message | jest, cypress |

### Steps to manually verify this change

1. Sign in as user
2. Create an APD
3. Confirm message shows
4a. Can dismiss message _or_
4b. Dismisses itself after a few minutes

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
